trieste/models/gpflow/models.py

```diff
@@ -1362,3 +1368,305 @@ def covariance_between_points(
             query_points_2=query_points_2,
             whiten=whiten,
         )
+
+
+class AR1(TrainableProbabilisticModel):
+    r"""
+    A :class:`TrainableProbabilisticModel` implementation of the AR1 model
+    from :cite:`Kennedy2000`. This is a multi-fidelity model that works with an
+    arbitrary number of fidelities. It relies on there being a linear relationship
+    between fidelities, and may not perform well for more complex relationships.
+    Precisely, it models the relationship between sequential fidelities as
+
+    .. math:: f_{i}(x) = \rho f_{i-1}(x) + \delta(x)
+
+    where :math:`\rho` is a scalar and :math:`\delta` models the residual between the fidelities.
+    All the base models used in this implementation are :class:`~gpflow.models.GPR` models.
+    Note: Currently only supports single output problems.
```

Comment:
does it support anything else than GPR?

Reply:
I'm not sure. I've only tried it with GPRs so far, and I think that's what the paper uses. I think because we have to do a reimplementation of the loss function we may be limited to GPRs, but @alansaul probably has more idea.

Reply:
possibly good enough for this PR but indeed it could be really nice to have an SGPR / SVGP implementation?

Reply:
Its fine if its restricted to GPR but make it clear in the docs